Finmer.Game/App.xaml.cs

```diff
@@ -55,6 +55,9 @@ protected override void OnStartup(StartupEventArgs e)
 
         protected override void OnExit(ExitEventArgs e)
         {
+            // Ensure all gameplay state is released while the app framework is still available
+            GameController.ExitSession();
```

Comment:
This branch includes commits from PR #5. Could you try cleaning up your version history here, then force-push up? I think it'd be the cleanest to hard-reset your master branch to the upstream HEAD (https://stackoverflow.com/a/42332860), that will help prevent you from needing to add those merge commits in the future, and then either rebase the last few commits on top of it, or just re-commit it (it's only a few lines after all).

In general, try to avoid committing to master, because this will force you to use merge commits to pull in my new (unrelated) changes; if you use only branches (and there's no need to merge in master there either) then you can always fast-forward the master branch.